README.md

```diff
@@ -22,7 +22,14 @@ A couple of notes:
 
 ## Creating the Graph
 
-The `umls-to-graph` code uses Maven to handle dependencies (see the `pom.xml` file for details). Run the code from Eclipse or IntelliJ, or compile a jar using Maven.
+The `umls-to-graph` code uses Gradle to handle dependencies (see the `build.gradle.kts` file for details). Run the code from Eclipse or IntelliJ, or compile a jar:
+
+```
+./gradlew assemble
```

Comment:
this is the default task that's set up for "do whatever tasks are necessary to produce the build artifacts that you might publish or otherwise use directly". The key task running under the hood is the `shadowJar` task, provided by the `shadow` plugin.